.github/workflows/deploy.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
               --namespace ${CONNECTORS_NS} \
               --from-file="${CONNECTORS_DIR}/${CM_NAME}/" \
               --dry-run=client \
-              -o yaml | kubectl apply -f -
+              -o yaml | kubectl replace -f -
```

Comment:
This will replace the entire ConfigMap instead of merging (thus `timer_source_0.1.json` wasn't removed)